(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> ##############
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> ##########
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> ############
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> ##########
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> python manage.py makemigrations images
Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    main()
  File "manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "C:\Users\romanpc\dev\env\env\bookmarks\lib\site-packages\django\core\management\__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "C:\Users\romanpc\dev\env\env\bookmarks\lib\site-packages\django\core\management\__init__.py", line 377, in execute
    django.setup()
  File "C:\Users\romanpc\dev\env\env\bookmarks\lib\site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Users\romanpc\dev\env\env\bookmarks\lib\site-packages\django\apps\registry.py", line 114, in populate
    app_config.import_models()
  File "C:\Users\romanpc\dev\env\env\bookmarks\lib\site-packages\django\apps\config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "C:\Python38\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 779, in exec_module
  File "<frozen importlib._bootstrap_external>", line 916, in get_code
  File "<frozen importlib._bootstrap_external>", line 846, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\romanpc\dev\env\env\bookmarks\bookmarks\images\models.py", line 27
    def __str__(self):
                     ^
TabError: inconsistent use of tabs and spaces in indentation
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> python manage.py makemigrations images
Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    main()
  File "manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "C:\Users\romanpc\dev\env\env\bookmarks\lib\site-packages\django\core\management\__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "C:\Users\romanpc\dev\env\env\bookmarks\lib\site-packages\django\core\management\__init__.py", line 377, in execute
    django.setup()
  File "C:\Users\romanpc\dev\env\env\bookmarks\lib\site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Users\romanpc\dev\env\env\bookmarks\lib\site-packages\django\apps\registry.py", line 114, in populate
    app_config.import_models()
  File "C:\Users\romanpc\dev\env\env\bookmarks\lib\site-packages\django\apps\config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "C:\Python38\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 779, in exec_module
  File "<frozen importlib._bootstrap_external>", line 916, in get_code
  File "<frozen importlib._bootstrap_external>", line 846, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\romanpc\dev\env\env\bookmarks\bookmarks\images\models.py", line 23
    users_like = models.ManyToManyField(settings.AUTH_USER_MODEL,
                                                                ^
TabError: inconsistent use of tabs and spaces in indentation
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> python manage.py makemigrations images
Migrations for 'images':
  images\migrations\0001_initial.py
    - Create model Image
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> python manage.py migrate images
Operations to perform:
  Apply all migrations: images
Running migrations:
  Applying images.0001_initial... OK
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> python manage.py runserver_plus --cert-file cert.crt
 * Restarting with stat
Performing system checks...

System check identified no issues (0 silenced).

Django version 3.1.2, using settings 'bookmarks.settings'
Development server is running at https://[127.0.0.1]:8000/
Using the Werkzeug debugger (http://werkzeug.pocoo.org/)
Quit the server with CTRL-BREAK.
 * Debugger is active!
 * Debugger PIN: 232-097-404
 * Running on https://127.0.0.1:8000/ (Press CTRL+C to quit)
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /admin HTTP/1.1" 301 -
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /admin/ HTTP/1.1" 302 -
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /static/admin/css/base.css HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /static/admin/css/login.css HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /static/admin/css/responsive.css HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /static/admin/css/fonts.css HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 20:28:39] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 -
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> pip install
ERROR: You must give at least one requirement to install (see "pip help install")
WARNING: You are using pip version 20.1.1; however, version 20.2.4 is available.
You should consider upgrading via the 'c:\users\romanpc\dev\env\env\bookmarks\scripts\python.exe -m pip install --upgrade pip' command.
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> pip install certifi
Requirement already satisfied: certifi in c:\users\romanpc\dev\env\env\bookmarks\lib\site-packages (2020.6.20)
WARNING: You are using pip version 20.1.1; however, version 20.2.4 is available.
You should consider upgrading via the 'c:\users\romanpc\dev\env\env\bookmarks\scripts\python.exe -m pip install --upgrade pip' command.
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> pip upgrade certifi
ERROR: unknown command "upgrade"
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> python manage.py runserver_plus --cert-file cert.crt
 * Restarting with stat
Performing system checks...

System check identified no issues (0 silenced).

Django version 3.1.2, using settings 'bookmarks.settings'
Development server is running at https://[127.0.0.1]:8000/
Using the Werkzeug debugger (http://werkzeug.pocoo.org/)
Quit the server with CTRL-BREAK.
 * Debugger is active!
 * Debugger PIN: 232-097-404
 * Running on https://127.0.0.1:8000/ (Press CTRL+C to quit)
Not Found: /images
127.0.0.1 - - [18/Oct/2020 20:51:12] "GET /images HTTP/1.1" 404 -
Not Found: /images/create
127.0.0.1 - - [18/Oct/2020 20:51:20] "GET /images/create HTTP/1.1" 404 -
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> python manage.py runserver_plus --cert-file cert.crt
 * Restarting with stat
Performing system checks...

System check identified no issues (0 silenced).

Django version 3.1.2, using settings 'bookmarks.settings'
Development server is running at https://[127.0.0.1]:8000/
Using the Werkzeug debugger (http://werkzeug.pocoo.org/)
Quit the server with CTRL-BREAK.
 * Debugger is active!
 * Debugger PIN: 232-097-404
 * Running on https://127.0.0.1:8000/ (Press CTRL+C to quit)
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> python manage.py runserver_plus --cert-file cert.crt
 * Restarting with stat
Performing system checks...

System check identified no issues (0 silenced).

Django version 3.1.2, using settings 'bookmarks.settings'
Development server is running at https://[127.0.0.1]:8000/
Using the Werkzeug debugger (http://werkzeug.pocoo.org/)
Quit the server with CTRL-BREAK.
 * Debugger is active!
 * Debugger PIN: 232-097-404
 * Running on https://127.0.0.1:8000/ (Press CTRL+C to quit)
127.0.0.1 - - [18/Oct/2020 21:08:02] "GET /account HTTP/1.1" 301 -
127.0.0.1 - - [18/Oct/2020 21:08:02] "GET /account/ HTTP/1.1" 302 -
127.0.0.1 - - [18/Oct/2020 21:08:02] "GET /account/login/?next=/account/ HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 21:08:03] "GET /static/css/base.css HTTP/1.1" 200 -
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> python manage.py runserver_plus --cert-file cert.crt
 * Restarting with stat
Performing system checks...

System check identified no issues (0 silenced).

Django version 3.1.2, using settings 'bookmarks.settings'
Development server is running at https://[127.0.0.1]:8000/
Using the Werkzeug debugger (http://werkzeug.pocoo.org/)
Quit the server with CTRL-BREAK.
 * Debugger is active!
 * Debugger PIN: 232-097-404
 * Running on https://127.0.0.1:8000/ (Press CTRL+C to quit)
127.0.0.1 - - [18/Oct/2020 21:14:41] "GET /account/ HTTP/1.1" 302 -
127.0.0.1 - - [18/Oct/2020 21:14:42] "GET /account/login/?next=/account/ HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 21:14:42] "GET /static/css/base.css HTTP/1.1" 200 -
127.0.0.1 - - [18/Oct/2020 21:14:45] "GET /account/login/?next=/account/ HTTP/1.1" 200 -
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> pip install easy-thumbnails==2.7
Collecting easy-thumbnails==2.7
  Downloading easy-thumbnails-2.7.tar.gz (66 kB)
     |████████████████████████████████| 66 kB 781 kB/s
Requirement already satisfied: django<4.0,>=1.11 in c:\users\romanpc\dev\env\env\bookmarks\lib\site-packages (from easy-thumbnails==2.7) (3.1.2)
Requirement already satisfied: pillow in c:\users\romanpc\dev\env\env\bookmarks\lib\site-packages (from easy-thumbnails==2.7) (7.0.0)
Requirement already satisfied: pytz in c:\users\romanpc\dev\env\env\bookmarks\lib\site-packages (from django<4.0,>=1.11->easy-thumbnails==2.7) (2020.1)
Requirement already satisfied: sqlparse>=0.2.2 in c:\users\romanpc\dev\env\env\bookmarks\lib\site-packages (from django<4.0,>=1.11->easy-thumbnails==2.7) (0.4.1)
Requirement already satisfied: asgiref~=3.2.10 in c:\users\romanpc\dev\env\env\bookmarks\lib\site-packages (from django<4.0,>=1.11->easy-thumbnails==2.7) (3.2.10)
Using legacy setup.py install for easy-thumbnails, since package 'wheel' is not installed.
Installing collected packages: easy-thumbnails
    Running setup.py install for easy-thumbnails ... done
Successfully installed easy-thumbnails-2.7
WARNING: You are using pip version 20.1.1; however, version 20.2.4 is available.
You should consider upgrading via the 'c:\users\romanpc\dev\env\env\bookmarks\scripts\python.exe -m pip install --upgrade pip' command.
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks> python manage.py migrate
Operations to perform:
  Apply all migrations: account, admin, auth, contenttypes, easy_thumbnails, images, sessions, social_django
Running migrations:
  Applying easy_thumbnails.0001_initial... OK
  Applying easy_thumbnails.0002_thumbnaildimensions... OK
(bookmarks) PS C:\Users\romanpc\dev\env\env\bookmarks\bookmarks>
